source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+| ``module update``              | Reloads all currently loaded modules       |
++--------------------------------+--------------------------------------------+
+
+Searching for Modules
+---------------------
 
-Data transfers between Gaea and the world outside of Gaea should be
-performed on the Remote Data Transfer Nodes (RDTNs). There is a
-NOAA-developed tool called **gcp**, which is available for data
-transfers on Gaea. HPSS users are only able to access HPSS from jobs
-on the RDTNs. To tell a job script to run on the login nodes, specify
-the following in your script:
+Modules with dependencies are only available when the underlying dependencies,
+such as compiler families, are loaded. Thus, module avail will only display
+modules that are compatible with the current state of the environment. To
+search the entire hierarchy across all possible dependencies, the ``spider``
+sub-command can be used as summarized in the following table.
+
++-----------------------------------------+-----------------------------------+
+| Command                                 | Description                       |
++=========================================+===================================+
+| ``module spider``                       | Shows the entire possible graph   |
+|                                         | of modules                        |
++-----------------------------------------+-----------------------------------+
+| ``module spider <modulename>``          | Searches for modules named        |
+|                                         | ``<modulename>`` in the graph of  |
+|                                         | possible modules                  |
++-----------------------------------------+-----------------------------------+
+|``module spider <modulename>/<version>`` | Searches for a specific version   |
+|                                         | of ``<modulename>`` in the graph  |
+|                                         | of possible modules               |
++-----------------------------------------+-----------------------------------+
+| ``module spider <string>``              | Searches for modulefiles          |
+|                                         | containing ``<string>``           |
++-----------------------------------------+-----------------------------------+
 
-.. code-block:: shell
 
-  #SBATCH --clusters=es
-  #SBATCH --partition=rdtn_c#
-  #SBATCH --nodes=1
-  #SBATCH --ntasks-per-node=1 #set ntasks-per-node to the number of cores your job will need, up to 8
+Compilers
+=========
 
-or, from the sbatch command line:
+Cray, AMD, NVIDIA, and GCC compilers are provided through modules on Gaea.
+There is also a system/OS versions of GCC available in :file:`/usr/bin`. The
+table below lists details about each of the module-provided compilers. Please
+see the following :ref:`gaea-compiling` section for more detailed inforation on
+how to compile using these modules.
```

Comment:
See the following :ref:`gaea-compiling` section for more detailed information on using these modules to compile.

Reply:
Adjusted in. anew push.